### PR TITLE
Clean up withTranslation()

### PR DESCRIPTION
**Pull request by GabrielBruno24**
Many of the uses of withTranslation() had outdated arguments, either missing translation files or not including needed ones.

This PR updates them to correctly match the current content of the files. Also removes brackets in case of 1 file in the argument.

Also removes the instance of withTranslation() that have 0 arguments, updating some props types that extend WithTranslation along the way.

### CODE REVIEW

**Comment by greenscientist** [comment]
Definitely more a review for @tahini , maybe @kaligrafy . I have no idea what is going on :)

File: packages/chaire-lib-frontend/src/components/pageParts/Header.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ const UserMenu: React.FunctionComponent<UserMenuProps & WithTranslation> = (prop
         </div>
     );
 };
-const TranslatableUserMenu = withTranslation('menu')(UserMenu);
+const TranslatableUserMenu = withTranslation()(UserMenu);
```
Comment (tahini) [inline packages/chaire-lib-frontend/src/components/pageParts/Header.tsx:118]: Pourquoi laisser des parenthèses vides? Si on mets les namespaces, ne faudrait-il pas les mettre partout?

**Review by tahini — COMMENTED** [review]
Merci pour le nettoyage! J'ai juste un questionnement sur les valeurs laissées vides. Qu'est-ce que ça signifie? Est-ce des composants qui n'ont pas de traductions finalement? Si oui, on peut enlever le withTranslation complètement?

**Review by tahini — APPROVED** [review]
excellent merci!

**Review by kaligrafy — APPROVED** [review]

**Pull request merged**